deskpro/deskpro-admin-frontend
# Fix/8022 filter properties translate

Issue:
![issue](https://user-images.githubusercontent.com/46844842/72957218-146ef100-3da3-11ea-9110-35f3fd1970da.png)

Test Steps:
Go to "Agents" page and click on "Filter" button.

Expected behavior:
![fixed](https://user-images.githubusercontent.com/46844842/72957182-f73a2280-3da2-11ea-8bef-af1539a3fab9.png)
